### DISCUSSION TITLE
Closed Loop stepper motor and Driver combo with Open FFB Configurator

### DISCUSSION THREAD

**Discussion by johnujose** [Ultrawipf/OpenFFBoard#95] [Q&A] [answered]
Hi,

@Ultrawipf First of all I really appreciate the time and effort you have put into building this project. It really is a great one.
@willson556 @jonlidgard @M3gaFr3ak 
I was planning to build one for myself but it seems that I cannot get my hands on the Hardware that is designed in this project.
I paln to use a Closed loop stepper motor with a driver combo to achieve the same. Link given below.
https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.alibaba.com%2Fproduct-detail%2FHB860H-86HB250-156BJ-Nema-34-12_60784428647.html&psig=AOvVaw3nxy-Etsn-WlMm9FBoNryS&ust=1682696850408000&source=images&cd=vfe&ved=0CBMQjhxqFwoTCKjqt7K0yv4CFQAAAAAdAAAAABAE

So my question is that can I use the RS232 port on the driver to connect the PC and drive the motor using the Open FFB configurator?
Or is there any other application that can make this setup work with the PC games.

If not can I add a stm32 black pill (https://stm32-base.org/boards/STM32F103C8T6-Black-Pill.html) in between the PC and the stepper driver to generate the PWM step and direction signals? Will this setup work and how much of an effort will it be to port the current code to run on this setup.

Thanks and regards.
John

**Comment by Ultrawipf** [comment, marked as answer]
This is not a torque mode capable driver so you can not use it for FFB.
This seems to be just a stepper driver that can compensate for position errors.

You will need either a supported driver or one that can accept a torque setpoint via PWM
